specs/RuntimeSpec/Scenarios/FinancialTransaction.md

```diff
@@ -110,10 +110,11 @@ If any of the above operations fail all of the changes will be reverted.
 The receipt created in the previous section will eventually arrive to a runtime on the shard that hosts `bob_near` account.
 Again, it will be processed by `Runtime::apply` which will immediately call `Runtime::process_receipt`.
 It will check that this receipt does not have data dependencies (which is only the case of function calls) and will then call `Runtime::apply_action_receipt` on `TransferAction`.
+
 `Runtime::apply_action_receipt` will perform the following checks:
 
 - Retrieves the state of `bob_near` account, if it still exists (it is possible that Bob has deleted his account concurrently with the transfer transaction);
-- Applies the rent to Bob's account;
+- Checks if `bob_near` has enough balance for storage staking;
```

Comment:
this sounds a bit confusing to me
1. there is no change in storage, so the check is not needed for the example (of course it is still done in reality, just not very relevant to explain how a NEAR token transfer works)
2. In the code, the check is done after everything has been applied to the temporary state. The temporary state is dropped / rolled back if there is not enough balance. Maybe this level of detail is not relevant but still I would consider switching the order in this list, moving the storage staking check at the end.

Reply:
have updated section, I like to include detail myself, especially here considering it's Nomicon, but if you still feel this does not belong in this run-through I'm happy to remove!

Reply:
yeah with this item at the end, it's probably good to keep it in